<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sultan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="LibGUI::TabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSetting"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuAdministration">
    <property name="title">
     <string>&amp;Administration</string>
    </property>
    <addaction name="action_User"/>
   </widget>
   <widget class="QMenu" name="menuInventory">
    <property name="title">
     <string>&amp;Inventory</string>
    </property>
    <addaction name="action_Category"/>
    <addaction name="action_Suplier"/>
    <addaction name="action_Items"/>
   </widget>
   <widget class="QMenu" name="menuTransaction">
    <property name="title">
     <string>&amp;Transaction</string>
    </property>
    <addaction name="action_Cashier"/>
    <addaction name="action_Purchase"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdministration"/>
   <addaction name="menuInventory"/>
   <addaction name="menuTransaction"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSetting">
   <property name="text">
    <string>&amp;Setting</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>&amp;Logout</string>
   </property>
  </action>
  <action name="action_User">
   <property name="text">
    <string>&amp;User</string>
   </property>
  </action>
  <action name="action_Suplier">
   <property name="text">
    <string>&amp;Suplier</string>
   </property>
  </action>
  <action name="action_Items">
   <property name="text">
    <string>&amp;Items</string>
   </property>
  </action>
  <action name="action_Cashier">
   <property name="text">
    <string>&amp;Cashier</string>
   </property>
  </action>
  <action name="action_Purchase">
   <property name="text">
    <string>&amp;Purchase</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_Category">
   <property name="text">
    <string>&amp;Category</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LibGUI::TabWidget</class>
   <extends>QTabWidget</extends>
   <header>tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
